docs/pages/kubernetes-access/discovery/azure.mdx

```diff
@@ -233,7 +233,7 @@ teleport:
     token_name: "/tmp/token"
     method: token
   auth_servers:
-  - "teleport.example.com:3080"
+  - "teleport.example.com:443"
```

Comment:
Let's update this to use config v3 while we're here. It's confusing to users to specify a proxy server under `auth_servers`.